### ISSUE TITLE
Go to previous or forward edit

### ISSUE THREAD

**Issue by brunolemos** [plrenaudin/goto-last-edit#3]
Very used in WebStorm. Example:

`Cmd + [` go one item to the left in the history
`Cmd + ]` go one item to the right in the history

**Comment by plrenaudin** [comment]
This is fixed but not published yet.
The shortcut are already taken for indent/unindent though.
Will publish the change tomorrow

**Comment by plrenaudin** [comment]
Done and available in 0.3.0

**Comment by brunolemos** [comment]
Thanks, this is working fine without conflict:

```
'atom-workspace atom-text-editor:not([mini])':
  'cmd-{':        'goto-last-edit:back'
  'cmd-}':        'goto-last-edit:forward'
  'cmd-[':        'cursor-history:prev'
  'cmd-]':        'cursor-history:next'
  'ctrl-cmd-[':   'cursor-history:prev-within-editor'
  'ctrl-cmd-]':   'cursor-history:next-within-editor'
```
